gadget/validate_test.go

```diff
@@ -1020,3 +1035,154 @@ assets:
 	err = gadget.ValidateContent(ginfo, s.dir, kernelUnpackDir)
 	c.Assert(err, ErrorMatches, `no asset from the kernel.yaml needing synced update is consumed by the gadget at "/.*"`)
 }
+
+func (s *validateGadgetTestSuite) TestValidateModalClassicGadget(c *C) {
```

Comment:
we probably don't want to use modal naming also here

Reply:
oops, I overlooked these ones, changed now